src/components/Homepage/Homepage.js

```diff
@@ -1,22 +1,40 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import 'balloon-css'; 
 
 function Homepage() {
   return (
     <div className='homepage'>
-      <h2 className='home-title'>What Do You Want To Do?</h2>
+      <h2 className='home-title'>What do you want to do?</h2>
       <Link to={'/addjob'}>
-        <button className='home add'>Add Job</button>
+        <button className='home add'>Add Job
+          <br /><i className="fas fa-tools"></i>
+          </button>
       </Link>
       <Link to={'/eligiblejobs/grace-period'}>
-        <button className='home eligible'>NOI Eligible Jobs</button>
+        <button className='home eligible'>NOI Eligible Jobs
+          <br /><i className="far fa-calendar-check"></i>
+        </button>
       </Link>
       <Link to={'/filedjobs/lien-eligible'}>
-        <button className='home filed'>Filed Liens</button>
+        <button className='home filed'>Filed Liens
+          <br /><i className="far fa-file-alt"></i>
+        </button>
       </Link>
       <Link to={'/profile'}>
-        <button className='home profile'>Profile</button>
+        <button className='home profile'>Profile
+          <br /><i className="fas fa-user-alt"></i>
+        </button>
       </Link>
+      <button
+        data-balloon-length='large'
```

Comment:
This looks great in practice. We should definitely look over what it says and see if we can have more clarity over our explanation